nmdc_runtime/api/core/util.py

```diff
@@ -28,10 +28,13 @@ def hash_from_str(s: str, algo="sha256") -> str:
     return getattr(hashlib, algo)(s.encode("utf-8")).hexdigest()
 
 
-def sha256hash_from_file(file_path: str):
+def sha256hash_from_file(file_path: str, timestamp: str):
     # https://stackoverflow.com/a/55542529
     h = hashlib.sha256()
 
+    timestamp_bytes = timestamp.encode('utf-8')
+    h.update(timestamp_bytes)
```

Comment:
This change makes sense to me.

[Docs](https://docs.python.org/3/library/stdtypes.html#str.encode) for `str.encode()` say that it returns the `bytes` equivalent of the string.

[Docs](https://docs.python.org/3/library/hashlib.html#hashlib.hash.update) for `hash.update()` say that it adds `bytes` to the hash (and that calling it multiple times is like concatenating the individual arguments and calling it once with the concatenation result).